app/src/main/java/com/github/libretube/helpers/PlayerHelper.kt

```diff
@@ -343,6 +344,11 @@ object PlayerHelper {
             )
 
     fun shouldPlayNextVideo(isPlaylist: Boolean = false): Boolean {
+        // if there is no next video, it obviously should not be played
+        if (PlayingQueue.getNext() == null) {
```

Comment:
```suggestion
        if (!PlayingQueue.hasNext()) {
```

Reply:
I'm not a fan of this style, as I think it makes the code harder to read, but if you prefer it that way…